data/features.yml

```diff
@@ -401,8 +401,8 @@ server_side_batch_changes:
 code_insights:
```

Comment:
Is code insights still one product area with just one feature, or does it make sense yet to break this out into different capabilities?

Reply:
I think it makes sense to keep this as one unified thing for now, but thanks for highlighting that's an option in the future! (Was there something you felt like we should break out?)

Reply:
No I didn't - just noticed that it was one thing and wanted to ask. If it makes sense to you, then it's correct.